buildenv/jenkins/common/build.groovy

```diff
@@ -572,7 +572,21 @@ def build_all() {
                 timeout(time: 5, unit: 'HOURS') {
                     try {
                         // Cleanup in case an old build left anything behind
-                        cleanWs disableDeferredWipeout: true, deleteDirs: true
+                        // cleanWS doesn't always work on Windows #10834
+                        try {
+                                if (SPEC.contains('win')) {
+                                        if (WORKSPACE != null && !WORKSPACE.isEmpty()) {
+                                                buildWorkspace = sh(script: "cygpath -u '${env.WORKSPACE}'", returnStdout: true).trim()
+                                                if (buildWorkspace != null) {
+                                                        sh "rm -fr ${buildWorkspace}/*"
+                                                }
+                                        }
+                                } else {
+                                        cleanWs disableDeferredWipeout: true, deleteDirs: true
+                                }
+                        } catch (e) {
```

Comment:
I don't think this try/catch has any value.

Reply:
try needs either a catch or a finally.